frontend/src/components/DocumentDetails.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import { Typography, Button, withStyles} from 'material-ui';
+import FileDownload from 'material-ui-icons/FileDownload';
+import classNames from 'classnames';
+
+import moment from 'moment';
+import 'moment/locale/fr';
+
+import { getDocument } from '../api';
+import PdfPreview from './PdfPreview';
+import ImagePreview from './ImagePreview';
+import VideoPreview from './VideoPreview';
+
+const styles = theme => ({
+  floatRight: {
+    float: 'right'
+  },
+  preview: {
```

Comment:
`preview`, `progress`, `stepper` and `page` have been moved to PdfPreview, but left here ^^